packages/cli/package.json

```diff
@@ -6,10 +6,8 @@
   "bin": "./bin/run",
   "bugs": "https://github.com/heroku/cli/issues",
   "dependencies": {
-    "@heroku/buildpack-registry": "^1.0.1",
```

Comment:
All the changes in here are due to sorting alphabetically, except `"@oclif/core": "^2.8.11"` & `"@heroku-cli/command": "^10.0.0"`